### PR TITLE
Add cargo and clean targets to Makefile

### PR DESCRIPTION
**Pull request by pythcoiner**
This PR add `cargo` and `clean` targets to Makefile.

### CODE REVIEW

**Comment by brunoerg** [comment]
Thanks for it, will review soon!

**Review by brunoerg — CHANGES_REQUESTED** [review]
Can you remove this Makefile.kate-swp?

File: Makefile [hunk 1]
```diff
@@ -1,5 +1,13 @@
 HEADERS :=  $(wildcard $(shell find bitcoin -type f -name '*.h')) compiler.h targets/miniscript_policy.h targets/miniscript_string.h targets/block_des.h targets/prefilledtransaction.h
 SOURCES :=  $(wildcard $(shell find bitcoin -type f -name '*.cpp')) compiler.cpp targets/miniscript_policy.cpp targets/miniscript_string.cpp targets/block_des.cpp targets/prefilledtransaction.cpp
 
-bitcoinfuzz: $(HEADERS) $(SOURCES) fuzzer.cpp
-	clang++ -O3 -g0 -Wall -fsanitize=address,fuzzer -DHAVE_GMTIME_R=1 -std=c++20 -march=native -L rust_bitcoin_lib/target/release  -lrust_bitcoin_lib -lpthread -ldl -flto -Ibitcoin $(SOURCES) fuzzer.cpp -o bitcoinfuzz
+bitcoinfuzz: cargo $(HEADERS) $(SOURCES) fuzzer.cpp
+	echo "Running clang++"
```
Comment (brunoerg) [inline Makefile:5]: I think you could remove this echo because the clang++ command is printed.

**Comment by pythcoiner** [comment]
all done

**Pull request merged**